atta/controller/atta_cli.py

```diff
@@ -2,7 +2,7 @@
 import click
 
 @click.command()
-@click.option('--readcsv', default="readcsv.csv", help='read csv format data')
+@click.option('--readcsv', default="data.csv", help='read csv format data')
```

Comment:
Why do you change this name?

Reply:
I thought with the name "data.csv" is more clear to indicate this is just the data file.

Reply:
This is good idea. However the other part need to be changed in the meantime. Like def main(readcsv). Would you please follow README and make sure `atta --data xxx.csv` still work after you change the click.option and its associated code? Then we could call "the new algorithm is integrated into the original comment line tool".

Reply:
I think you misunderstand the change. This only change the default file name, not the command line command. Therefore it still work as atta --readcsv xxx.csv
I have tested it and it worked.
Nevertheless, if you like to keep the original default filename as readcsv.csv, it is also fine with me.

Reply:
Right. Thanks for the elaboration. \o/